pom.xml

```diff
@@ -111,6 +117,10 @@
             <url>https://repository.jboss.org/nexus/content/groups/ea</url>
         </repository>
         <repository>
+            <id>fuse-early-access-repository origin</id>
+            <url>https://origin-repository.jboss.org/nexus/content/groups/ea</url>
+        </repository>
```

Comment:
I think this is not correct. Now we have both `https://repository.jboss.org/nexus/content/groups/ea` and `https://origin-repository.jboss.org/nexus/content/groups/ea`